### PR TITLE
Bugfix | correct api call on other services screen

### PR DESCRIPTION
**Pull request by omarCreativeDev**

### CODE REVIEW

**Comment by omarCreativeDev** [comment]
[(1) Bulk Upload - Passed Validation with No Warnings](https://trello.com/c/B7rG0kit/69-1-bulk-upload-passed-validation-with-no-warnings)

**Comment by omarCreativeDev** [comment]
[(2) Main service is being displayed and selected in the list of other services](https://trello.com/c/d5M534JQ/76-2-main-service-is-being-displayed-and-selected-in-the-list-of-other-services)

**Comment by omarCreativeDev** [comment]
[(2) Regression Issue: Incorrect list of 'other services' is displayed for CQC-registered establishment](https://trello.com/c/HKBdmPfM/77-2-regression-issue-incorrect-list-of-other-services-is-displayed-for-cqc-registered-establishment)

File: src/app/core/services/workplace.service.ts [hunk 1]
```diff
@@ -18,4 +18,11 @@ export class WorkplaceService {
   public getServicesByCategory(isRegulated: boolean): Observable<Array<ServiceGroup>> {
     return this.http.get<Array<ServiceGroup>>(`/api/services/byCategory?cqc=${isRegulated}`);
   }
+
+  public getAllServices(establishmentId): Observable<ServiceGroup[]> {
```
Comment (ChazUK) [inline src/app/core/services/workplace.service.ts]: if this is related to the Establishment I think it belongs in the EstablishmentService

File: src/app/features/workplace/other-services/other-services.component.html [hunk 2]
```diff
@@ -30,7 +30,7 @@ <h2 class="govuk-fieldset__heading">
               name="otherServices"
               type="checkbox"
               [value]="service.id"
-              [checked]="service.other"
+              [checked]="service.isMyService"
```
Comment (ChazUK) [inline src/app/features/workplace/other-services/other-services.component.html]: I updated this in my code to remove the dependency on that list of services returned from the API.

```
[checked]="form.get('otherServices').value.includes(service.id)"
```

Comment (omarCreativeDev) [reply src/app/features/workplace/other-services/other-services.component.html]: cheers, just fixed

File: src/app/core/services/workplace.service.ts (hunk 1 above)
Comment (omarCreativeDev) [reply src/app/core/services/workplace.service.ts]: cheers, just fixed

**Pull request merged**